src/main/java/org/junitpioneer/jupiter/issue/IssueExtensionExecutionListener.java

```diff
@@ -37,7 +36,7 @@
  */
 public class IssueExtensionExecutionListener implements TestExecutionListener {
 
-	public static final String REPORT_ENTRY_KEY = "Issue";
+	public static final String REPORT_ENTRY_KEY = IssueExtensionExecutionListener.class.toString();
```

Comment:
Hmm. We define the report key in all extensions etc. with a string and not by the name of a class. Don't know if I like this change

Reply:
I don't _really_ like it either, but I think "Issue" is just a tad bit too generic. 🤷 

Perhaps this?
```
public static final String REPORT_ENTRY_KEY = "IssueExtensionExecutionListener";
```

Reply:
I like this better, so we can refactor (if needed) the class without breaking "the API".